pkg/api/index.go

```diff
@@ -621,7 +621,7 @@ func (hs *HTTPServer) setIndexViewData(c *models.ReqContext) (*dtos.IndexViewDat
 		Sentry:                  &hs.Cfg.Sentry,
 		Nonce:                   c.RequestNonce,
 		ContentDeliveryURL:      hs.Cfg.GetContentDeliveryURL(hs.License.ContentDeliveryPrefix()),
-		LoadingLogo:             "public/img/grafana_icon.svg",
```

Comment:
1. I'm concerned that we'll upgrade to a new version of Grafana and miss a new usage of `grafana_icon.svg`. I think we should modify the contents of the `svg` itself to either contain the NI logo, or our standard placeholder graphic from UX. The latter is probably less risky of running afoul of the brand policies for using the icon.
2. We should make these same modifications to the other Grafana logos in `public/img`. There seem to be different variants of the logo depending on the theme, for example.